tests/unittests/config/test_schema.py

```diff
@@ -53,6 +55,17 @@
 
 M_PATH = "cloudinit.config.schema."
 
+INSTANCE_ID = "i-testing"
+
+
+class FakeDataSource(sources.DataSource):
```

Comment:
Nit (not something that needs to be changed): I'd like to consolidate this type of helper class for tests into `unittests/util.py`. We already have a `DataSourceTesting` that is more for testing implementations that require using `get_cloud`, but I think the implementation here (or perhaps a helper function) could live there too.

Reply:
Done.